Fallout4VR/Fallout4VRGameModeDescriptor.cs

```diff
@@ -10,8 +10,8 @@ public class Fallout4VRGameModeDescriptor : GamebryoGameModeDescriptorBase
 	{
 		private static string[] EXECUTABLES = { "Fallout4VR.exe" };
 		private static string[] CRITICAL_PLUGINS = { "Fallout4.esm", "Fallout4_VR.esm" };
-		private static string[] OFFICIAL_PLUGINS = { };
-        private static string[] OFFICIAL_UNMANAGED_PLUGINS = { };
+        private static string[] OFFICIAL_PLUGINS = { "DLCRobot.esm", "DLCworkshop01.esm", "DLCCoast.esm", "DLCworkshop02.esm", "DLCworkshop03.esm", "DLCNukaWorld.esm" };
```

Comment:
On second thought, this might not be entirely correct? Technically these aren't official plugins for Fallout 4 VR, and NMM will still handle the files if they're put in the FO4VR data folder. ¯\_(ツ)_/¯